src/Compilers/VisualBasic/Portable/Binding/Binder_Delegates.vb

```diff
@@ -134,6 +134,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                                 ' Insert an identity conversion if necessary.
                                 Debug.Assert(boundFirstArgument.Kind <> BoundKind.Conversion, "Associated wrong node with conversion?")
                                 boundFirstArgument = New BoundConversion(node, boundFirstArgument, ConversionKind.Identity, CheckOverflow, True, delegateType)
+                            ElseIf boundFirstArgument.Kind = BoundKind.Conversion Then
+                                Debug.Assert(Not boundFirstArgument.WasCompilerGenerated)
+                                Dim boundConversion = DirectCast(boundFirstArgument, BoundConversion)
+                                boundFirstArgument = boundConversion.Update(boundConversion.Operand,
+                                                                            boundConversion.ConversionKind,
+                                                                            boundConversion.Checked,
+                                                                            True,
```

Comment:
>True, [](start = 76, length = 5)

Consider adding a comment at the end of this line with the name of the parameter, or using named argument.

Reply:
Comment added.

---
In reply to: [142805013](https://github.com/dotnet/roslyn/pull/22056#discussion_r142805013) [](ancestors = 142805013)